fitbenchmarking/fitbenchmarking
# Amend the workflow so that all commits go to master

I've used the workflow as described at
https://dojotoolkit.org/reference-guide/1.10/developer/github.html
which, of all the backporting discussions, seems to mirror ours
most.

#### Description of Work

Fixes #555 


#### Testing Instructions

1. read the docs -- check they make sense
2.
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
